classes/script_metadata.php

```diff
@@ -40,8 +40,14 @@ class script_metadata {
         manager::FLAME_OFF_PARAM_NAME,
     ];
 
-    /** List of paramteres that are to be recorded in redacted form. */
-    const REDACTLIST = ['sesskey'];
+    /** List of parameters that are to be recorded in redacted form. */
+    const REDACTLIST = [
```

Comment:
Constants should typically be in uppercase and separated by underscores for each word used.

In this case, it would probably be REDACT_LIST

https://docs.moodle.org/dev/Coding_style#Constants_2